pkg/ctl/cmdutils/configfile.go

```diff
@@ -426,6 +426,90 @@ func NewDeleteNodeGroupLoader(cmd *Cmd, ng *api.NodeGroup, ngFilter *NodeGroupFi
 	return l
 }
 
+// NewScaleNodeGroupLoader will load config or use flags for 'eksctl scale nodegroup'
+func NewScaleNodeGroupLoader(cmd *Cmd, ng *api.NodeGroup, ngFilter *NodeGroupFilter) ClusterConfigLoader {
```

Comment:
Can we move this to its own file?

Reply:
sure :100:

Reply:
what do you have in mind for file name ? I am thinking about group all node related loader into one file (e.g. nodegroup_configfile.go)

Or just specific file for scale node group e.g. scale_nodegroup_confilefile.go ?

Reply:
There are similar files in that package like `fargate.go` so perhaps `scale.go`

Reply:
thanks, I was a little bit confused with some other files like _create_cluster.go_. Let me rename it to scale.go :tada: